main_logic/tts_client.py

```
@@ -990,3 +1007,241 @@ def get_tts_worker(core_api_type='qwen', has_custom_voice=False):
+                break
+
+            if sid != current_speech_id:
+                current_speech_id = sid
+                resampler.clear()  # 新的一句，重置重采样状态
+                text_buffer = "" # 清空缓冲区域
+
+            if sid is None:
+                # 收到终止信号,可以在这里进行清理
+                # 例如 发送特殊的完成信息到服务器 或者重置状态
+                if text_buffer.strip():
+                    await send_text_to_server(ws, text_buffer)  # 封装发送逻辑
```

Ruff (0.14.10)

Undefined name `send_text_to_server`

(F821)